### PR TITLE
sudo: improve grammar

### PR DESCRIPTION
**Pull request by sbrl**
----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/sudo.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
-- To shutdown the machine:
+- Shut down the machine:
 
 `sudo {{shutdown}} -h +10 {{"Cya soon!"}}`
 
-- To repeat the last command as sudo:
+- Repeat the last command as sudo:
 
 `sudo {{!!}}`
```
Comment (agnivade) [inline pages/common/sudo.md]: Remove tokens from this one.

File: pages/common/sudo.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # sudo
 
-> Execute a command as another user.
+> Executes a single command as another user.
```
Comment (agnivade) [inline pages/common/sudo.md]: It can me multiple commands too right ? - `sudo cmdx && cmdy` ? Or does it run `cmdy` as the current user ?

Comment (sbrl) [reply pages/common/sudo.md]: The `&&` would be another command and get executed as the current user, yeah. You need to use `sh -c '...'` or escape them IIRC.

Comment (agnivade) [reply pages/common/sudo.md]: Ah right.

**Comment by sbrl** [comment]
Oops! Didn't realise that this was pending changes form my end :stuck_out_tongue:

**Pull request merged**